guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -2,31 +2,73 @@
+The push permission is `edit_products`.
+* To pull content, a user must have permissions as long as *unauthenticated pull* is set to off by default in the lifecycle environment.
+The pull permissions are `view_lifecycle_environments`, `view_products`, `view_content_views`.
 
 ifndef::orcharhino[]
 .Container registries on {SmartProxies}
-
 On {SmartProxies} with content, the https://github.com/Katello/smart_proxy_container_gateway[Container Gateway] {SmartProxy} plugin acts as the container registry.
 It caches authentication information from Katello and proxies incoming requests to Pulp.
 The Container Gateway is available by default on {SmartProxies} with content.
 endif::[]
 
-.Procedure
+.Limitations of pushing content to the {Project} container registry
+* You can only push content to the {ProjectServer} itself.
+If you need pushed content on {SmartProxies} as well, use {SmartProxy} syncing.
+* The pushed container name must contain only lowercase characters.
+* Unless pushed repositories are published in a content view version, they do not follow the Registry Name Pattern.
+This is to ensure that users can push and pull from the same path.
+* Users are required to push and pull from the same path.
+If a user uses the label-based schema, they pull using labels.
+If a user uses the ID-based schema, they pull using IDs.
 
-Logging in to the container registry:
+.Procedure
+. Logging in to the container registry:
++
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # podman login {foreman-example-com}
 ----
 
-Listing container images:
+. Listing container images:
++
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # podman search {foreman-example-com}/
 ----
 
-Pulling container images:
+. Pulling container images:
++
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # podman pull {foreman-example-com}/my-image:<optional_tag>
 ----
+
+Pushing container images to the {Project} container registry:
+
+* To indicate which organization, product, and repository the container image belongs to, include the organization and product in the container repository name.
+* All labels are composed of entirely lowercase characters regardless of actual label capitalization.
+* Pushes follow one of two formats:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# podman push _image_ _{foreman-example-com}_/_organization_label_/_product_label_/_repository_name_[:tag]
+# podman push _image_ _{foreman-example-com}_/id/<organization_id>/<product_id>/<repository_name>[:tag]
+----
+
+* An example of a complete push:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# podman push _d2c94e258dcb_ _{foreman-example-com}_/default_organization/test_product/hello-world:latest
+Getting image source signatures
+Copying blob ac28800ec8bb done   |
+Copying config d2c94e258d done   |
+Writing manifest to image destination
+----
+* A repository in Katello is created after the content is pushed.
```

Comment:
```suggestion
A repository in Katello is created after the content is pushed.
```
I think this would work better as a separate para, to conclude the section on pushing.